ext/openssl/ossl_x509store.c

```diff
@@ -329,7 +328,16 @@ ossl_x509store_set_trust(VALUE self, VALUE trust)
 static VALUE
 ossl_x509store_set_time(VALUE self, VALUE time)
 {
-    rb_iv_set(self, "@time", time);
+    X509_STORE *store;
+    X509_VERIFY_PARAM *param;
+
+    GetX509Store(self, store);
+#if HAVE_X509_STORE_GET0_PARAM
```

Comment:
This should be `#ifdef`. (Not incorrect, but recent Ruby versions that enable `-Wundef` will emit a warning)